nixos/modules/services/web-apps/shiori.nix

```diff
@@ -40,6 +40,7 @@ in {
       environment.SHIORI_DIR = "/var/lib/shiori";
 
       serviceConfig = {
+        ExecStartPre = "${package}/bin/shiori migrate";
```

Comment:
Reverted in 1.5.4 and also master is on 1.5.4 already via https://github.com/NixOS/nixpkgs/pull/213332.

https://github.com/go-shiori/shiori/releases/tag/v1.5.4